odra-casper/wasm-env/src/lib.rs

```diff
@@ -20,11 +20,3 @@ pub use wasm_contract_env::WasmContractEnv;
 #[cfg(all(target_arch = "wasm32", not(feature = "disable-allocator")))]
 #[allow(unused_imports)]
 use ink_allocator;
-
```

Comment:
Why is this not needed anymore?
Have you tried to compile with `disable-allocator`?